marxs/optics/test/test_polarization.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+
+from marxs.optics.polarization import polarization_vectors
+
+def test_random_polarization():
+	'''tests the angles to vector function for polarization
+	
+	This test makes sure that there is no obvious bias in a single direction.
+	*** NOTE *** It is possible, but extremely unlikely, for this test to fail by chance.
```

Comment:
One way to deal wit hthat is to set `np.random.seed(0)` (or some other number where we know that it works). Then the random number generator will always give the same random numbers, in other words, the test becomes exactly repeatable. However, I would recommend leaving it as it is for now - this way we test some new numbers every time and so we check that is works for a new number every time  ;-)

If we see this failing by chance more than once, we can still set the random number seed.